expression/expr_to_pb.go

```diff
@@ -373,14 +373,16 @@ func (pc PbConverter) canFuncBePushed(sf *ScalarFunction) bool {
 		ast.Log10,
 		ast.Exp,
 		ast.Pow,
-		ast.Sin,
-		ast.Asin,
-		ast.Cos,
-		ast.Acos,
-		ast.Tan,
-		ast.Atan,
-		ast.Atan2,
-		ast.Cot,
+		// Rust use the llvm math functions, which have different precision with Golang/MySQL(cmath)
+		// open the following switchers if we implement them in coprocessor via `cmath`
+		// ast.Sin,
```

Comment:
Will these functions cause the current test cases failed when cast can be pushed down?

Reply:
These incompatible are detected by https://github.com/tikv/copr-test/pull/122 after the CAST push-down. We can reopen them after we implement them in Coprocessor via `cmath` instead of llvm math functions.